hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/functional/TestHoodieSparkRollback.java

```diff
@@ -100,7 +102,7 @@ protected HoodieWriteConfig getConfigToTestMDTRollbacks(Boolean autoCommit) {
   }
 
   protected HoodieWriteConfig getConfigToTestMDTRollbacks(Boolean autoCommit, Boolean mdtEnable) {
-    return HoodieWriteConfig.newBuilder()
```

Comment:
if this entire class is meant for table version 6, can we rename the test class accordingly.

Reply:
Addressed